exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,9 +6,12 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (number < 0) return "negative";
+  else if ((number) => 0) return "positive";
+}
```

Comment:
See the comment I left in `F-logical-operators/exercise2.js` about the best way to format the code.